dbus_service.py

```diff
@@ -639,14 +639,34 @@ def set_dbus_values(self):
             logging.debug(f"Inverter #{self.pvinverternumber} Current (/Ac/Out/L1/I): {current}")
             logging.debug("---")
         else:
-            pre = "/Ac/" + self.pvinverterphase
```

Comment:
In my (personal) opinion there is no advantage to setting "/ac/" in a variable as it's more of a constant (in this case) and doesn't improve readability

Reply:
I agree. Although it has a touch of 'magic number' anti pattern, it increases readability.

Reply:
You're right, I changed that. (Magic numbers are not good :D )